src/PostgREST/Middleware.hs

```diff
@@ -48,6 +55,13 @@ runWithClaims conf eClaims app req =
       ]
       (toS $ "{\"message\":\""<>message<>"\"}")
 
+envVarToSql :: Text -> [(Text, Text)] -> [ByteString]
```

Comment:
Good idea to extract this into its own function. Maybe it makes more sense to put it in the QueryBuilder module, with return type `[SqlFragment]`?

Also `envVarToSql` sounds confusingly like unix environment variables. What about `setSqlVars` or something?

Adding a comment describing the motivation for the function would be nice, like how it's used for setting the role, jwt claims, and http request headers as GUCs.